akka-stream/src/main/scala/akka/stream/Graph.scala

```diff
@@ -26,5 +26,10 @@ trait Graph[+S <: Shape, +M] {
 
   def named(name: String): Graph[S, M] = withAttributes(Attributes.name(name))
```

Comment:
Ouch, `Source.addAttributes` calls `withAttributes` as well, I guess I should update that as well?